datalad/support/annexrepo.py

```diff
@@ -152,6 +156,12 @@ def _run_annex_command(self, annex_cmd, git_options=[], annex_options=[],
         debug = ['--debug'] if lgr.getEffectiveLevel() <= logging.DEBUG else []
         backend = ['--backend=%s' % backend] if backend else []
 
+        git_options = git_options[:] if git_options else []
+        annex_options = annex_options[:] if annex_options else []
```

Comment:
theoretically it wasn't needed here, since we didn't modify it, right? (but ok to leave in -- just trying to see if I am not missing anything)